cmd/federation-controller/main.go

```diff
@@ -89,9 +120,51 @@ func main() {
 		log.Fatalf("failed to parse configuration passed to the program arguments: %v", err)
 	}
 
+	ctrl.SetLogger(zap.New(zap.UseFlagOptions(&opts)))
+
+	mgr, err := ctrl.NewManager(ctrl.GetConfigOrDie(), ctrl.Options{
+		Scheme: scheme,
+		Metrics: server.Options{
+			BindAddress: metricsAddr,
+		},
+		HealthProbeBindAddress: probeAddr,
+		LeaderElection:         enableLeaderElection,
+		LeaderElectionID:       "80807133.federation.openshift-service-mesh.io",
+	})
+	if err != nil {
+		log.Errorf("unable to start manager: %s", err)
```

Comment:
I find it strange that the Kubebuilder scaffold includes this log message.

```suggestion
		log.Errorf("unable to create manager: %s", err)
```